flyteidl/protos/flyteidl/core/literals.proto

```diff
@@ -180,4 +180,7 @@ message RetryStrategy {
     // Number of retries. Retries will be consumed when the job fails with a recoverable error.
     // The number of retries must be less than or equals to 10.
     uint32 retries = 5;
+
+    // Delay between retries.
+    google.protobuf.Duration retry_delay = 6;
```

Comment:
shall we make this more elaborate - 
```
min_delay
max_delay
exponent
```

we can simply implement exponent of 1 for now and we can add other exponents later? we can help